assets/sass/components/_licence.scss

```diff
@@ -42,50 +46,51 @@ body {
   break-inside: avoid;
 }
 
-.section::before {
-  counter-increment: sectionCounter;
-  content: "\a" counter(sectionCounter) ". ";
-  white-space: pre;
+.section-with-heading {
+  break-inside: avoid;
+
+  .heading {
+    font-weight: bolder;
+  }
 }
 
 .section {
-  clear: left;
-  text-align: justify-all;
+  text-align: justify;
   break-inside: avoid;
```

Comment:
Thanks for tidying this - its been on my list to come to. What do you think to a pure-CSS footer? Worth pursuing? The Gotenberg footer feels a bit dirty, but it works for now. Something we can come back to I think?

Reply:
I havent looked at the footer to be honest! Although I sure it can be done easy enough